src/index.ts

```diff
@@ -45,6 +43,7 @@ export async function run (operation = createWebdriverIO) {
         .option('-t, --npm-tag <tag>', 'Which NPM version you like to install, e.g. @next', DEFAULT_NPM_TAG)
         .option('-y, --yes', 'will fill in all config defaults without prompting', false)
         .option('-d, --dev', 'Install all packages as into devDependencies', true)
+        .option('-s, --skipCLI', 'Skip installation of cli', false)
```

Comment:
Can you describe when someone would use this flag? Usually `npm init wdio@latest ./` is called when no WebdriverIO is setup. If someone already has a it installed they can just do `npx wdio config`

Reply:
In the original issue the create-wdio package was used in a monorepo setup in which you might have the CLI in a parent directory. Now you can prevent the installation of the packages that the CLI installs but could not prevent the CLI from being installed until now. Now you can do this:
```
projects
  - node_modules/@wdio/cli
  - project1
  - project2
```
when you run `npm init wdio .` in the project1 directory while the cli already exists in the parent directory, it will install the cli in project 1 as well.
using the skipCLI option, you can prevent any installations in which case you can use all the packages from the parent directory successfully.

Reply:
That's a great idea!

I think the following should also work, do you know?
```js
const cliInstalled = await import('@wdio/cli').then(() => true, () => false)
```

Reply:
It should, I wonder about the performance implication of using either of these approaches. In case `@wdio/cli` is installed using [`import-meta-resolve`](https://www.npmjs.com/package/import-meta-resolve) is a much more performant operation.